src/core/src/main/java/org/apache/jmeter/threads/ThreadGroup.java

```diff
@@ -361,7 +361,7 @@ public boolean stopThread(String threadName, boolean now) {
     }
 
     /**
-     * Hard Stop JMeterThread thrd and interrupt JVM Thread if interrupt is true
+     * Hard Stop JMeterThread third and interrupt JVM Thread if interrupt is true
```

Comment:
This is probably `thread` and not `third`. We could probably get rid of `thr*d` here at all.